### DISCUSSION TITLE
`AdjustContrast` does what commonly called gamma correction

### DISCUSSION THREAD

**Discussion by function2-llx** [Project-MONAI/MONAI#6027] [General]
Hello everyone,

Seems like what `AdjustContrast` transformation does is commonly called "gamma correction". The operation of contrast adjustment is commonly defined as multiplying some factor (possibly blending with (1-factor) * mean). The link provided in #114, torchvision implementation for [adjust_contrast](https://github.com/pytorch/vision/blob/5e8e2f125f140d1e908cf424a6a85cacad758125/torchvision/transforms/functional_tensor.py#L184-L198), [adjust_gamma](https://github.com/pytorch/vision/blob/5e8e2f125f140d1e908cf424a6a85cacad758125/torchvision/transforms/functional_tensor.py#L244-L261), and Pillow suggest the same naming convention. 

This can be confusing sometimes, e.g., in  #5535, when talking about `AdjustContrast` in MONAI, one may refer to `adjust_contrast` in torchvision (like [this comment](https://github.com/Project-MONAI/MONAI/pull/5535#issuecomment-1316886344)), while they do different stuffs.

Shall we consider renaming this? Or maybe we're supposed to used `ColorJitter` from torchvision in this case.

https://github.com/Project-MONAI/MONAI/blob/f5708ea89af86ca74f88e93dabb0ec94c4944ac1/monai/transforms/intensity/array.py#L801-L809

**Comment by KumoLiu** [comment]
Hi @function2-llx, we could already use PyTorch TorchVision transform based on the specified transform name and args. Here is an example. https://github.com/Project-MONAI/tutorials/blob/4d4c565c681fd7df0b01889c9d599b9b73e0b401/pathology/hovernet/training.py#L209-L216

For the naming convention, do you have some comments here @wyli?

**Comment by function2-llx** [comment]
@KumoLiu Thank you for the reply. Yes, we can use torchvision-based transform, though we cannot specify the probability (no `prob` parameter in [`RandTorchVisionD`](https://github.com/Project-MONAI/MONAI/blob/f4902b220538f02f3621b37c3e1f2067c830e0fe/monai/transforms/utility/dictionary.py#L1464). Of course, I can implement a simple wrapper when necessary, or combine with `RandLambda`), and it also supports 2D only.
